lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -524,6 +527,47 @@ def test_submission_history_xss(self):
         response = self.client.get(url)
         self.assertFalse('<script>' in response.content)
 
+    def test_submission_history_contents(self):
+        # log into a staff account
+        admin = AdminFactory.create()
+
+        self.client.login(username=admin.username, password='test')
+
+        usage_key = self.course_key.make_usage_key('problem', 'test-history')
```

Comment:
Question: line 534 above we use named args, line 540 below we use named args, why don't we use named args here?